### ISSUE TITLE
Operations requires a year in it's date field

### ISSUE THREAD

**Issue by jbrider** [APSIMInitiative/ApsimX#837]
In the old version of Apsim, the Operations model allowed you to specify a date that only had month and day which would allow you to run the operation on the same date each of each year in a multi year simulation.
Current version requires a year to be entered as part of the date field.

**Comment by zur003** [comment]
This could probably be done, but what would be the appropriate string format for such a date? Would "1-12" be the 12th of January or the first of December? Or would it need to be "12-Jan" and "1-Dec"? With the way the parsing is currently handled, the .Net framework helpfully inserts the current year, so it may be a bit tricky to distinguish "12-Jan" from "12-Jan-2016". For that matter, it might be parsing "12-Jan" as "1-Jan-2012". I'm not really sure. We could get around that problem by requiring only specific date formats, but thought would need to be given to cultural variations.

**Comment by jbrider** [comment]
I can't say that I have used the operations model extensively - I ran into the problem trying to convert an old apsim file to use in the new Apsim and ran into the issue. In that file the format was saves as "12-Jan" which seemed to be pretty consistent with the old manager standards. Maybe something as simple as a checkbox that will tell the model to ignore the year in the interface - therefore retaining the standard date formatting options? I'm not sure something like that will be clear in the interface though, which is one of the reasons I didn't just go ahead and implement it.
